src/fides/ctl/core/utils.py

```diff
@@ -138,6 +139,12 @@ def git_is_dirty(dir_to_check: str = ".") -> bool:
     Checks to see if the local repo has unstaged changes.
     Can also specify a directory to check.
     """
+    if not is_git_dir(".git"):
+        echo_red(
+            "This command must be run at the top-level of a git repo. Please try again."
```

Comment:
Instead of this, can we make this command succeed if you either:
1. Don't have `git` installed (e.g. on a container without git)
2. Aren't running from within a git repo

The git check is a useful safeguard if you are truly in a repo, but it shouldn't be a requirement to use `fides pull` in my mind. There will be cases where you want to pull YAML to use for other purposes, like monitoring, CI checks, etc

Reply:
@NevilleS ok, I'll figure it out! GitPython can be installed without git executable being present, so I can make this work (GitPython is a non-optional dependency for fides)